src/components/greeting.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const Greeting = () => {
+  const greetings = useSelector((state) => state.greetingReducer);
+
+  return (
+    <div>
+      {greetings.map((greet) => (
+        <div key={greet.id}>
+          <h1>{greet.text}</h1>
+        </div>
+      ))}
```

Comment:
- [ ] Only the fetched message from the API should be displayed, not all greetings. Kindly refactor your store and this component to resolve that.